CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## ToBeReleased
 
+## v4.4.1
```

Comment:
You should not add a new release version. Your items should be under the `ToBeReleased` section. Morevoer, your branch is not based with `master` anymore. Hence you have conflicts in your CHANGELOG file.

The new release version will be added by the person which releases a new version as specified in the README